These modifications/additions to VfilePattern allow the Fedora 20
installer to pull the files it needs via apt-cacher-ng.

(acng.conf needs to be updated with the new regex as well)

diff -ru apt-cacher-ng-0.7.26.orig/source/acfg_defaults.cc apt-cacher-ng-0.7.26/source/acfg_defaults.cc
--- apt-cacher-ng-0.7.26.orig/source/acfg_defaults.cc	2014-03-01 11:43:45.000000000 -0500
+++ apt-cacher-ng-0.7.26/source/acfg_defaults.cc	2014-04-13 17:47:01.005905617 -0400
@@ -41,7 +41,8 @@
 		"|filelists\\.xml\\.gz|filelists\\.sqlite\\.bz2|repomd\\.xml" // SL, http://ra.khe.sh/computers/linux/apt-cacher-ng-with-yum.html
 		"|packages\\.[a-zA-Z][a-zA-Z]\\.gz|info\\.txt|license\\.tar\\.gz|license\\.zip" //opensuse
 		"|"ALXPATTERN // Arch Linux
-		"|metalink\\?repo|.*prestodelta\\.xml\\.gz|repodata/.*\\.(xml|sqlite)"COMPRLIST // CentOS
+		"|metalink\\?repo|.*prestodelta\\.xml\\.gz|repodata/.*\\.(xml|sqlite)"COMPOPT // CentOS
+		"|\\.treeinfo|vmlinuz|(initrd|product|squashfs|updates)\\.img" // fedora
 		")$" // end of only-filename paterns
 		"|/dists/.*/installer-[^/]+/[^0-9][^/]+/images/.*"); // d-i stuff but not containing a number (year) in the revision directory (like "current", "beta", ...)
 
